### DISCUSSION TITLE
How does "Request Driver" work?

### DISCUSSION THREAD

**Discussion by Kozmito** [Courseplay/courseplay#7727] [Q&A] [answered]
Hello, I'm kinda new to Courseplay but I'm using a combine and I would like to know how does the command "Request Driver" work. 
Thank you.

**Comment by Tensuko** [comment, marked as answer]
If you drive the combine, it wont work.
Combine must be driven by cp.
It is just to call an unloader earlier.

**Reply by Kozmito** [reply]
I guess that the unloader is controlled by cp too right? Or by autodrive?

**Reply by Tensuko** [reply]
Yes, ofc by CP.

**Reply by Kozmito** [reply]
And how do i get a CP controlled unloader? That's other question but thank you anyways.
Edit: I mean the mode it has to be etc.

**Reply by Kozmito** [reply]
oh-nvm. i get it now.
Thank you so much!
